.flake8

```diff
@@ -1,4 +1,7 @@
 [flake8]
 # E501: Line length is enforced by Black, so flake8 doesn't need to check it
 # W503: Black disagrees with this rule, as does PEP 8; Black wins
-ignore = E501, W503
+# E402: Module level import not at top of file, due to importlib check
+# E701: Flake8 Catches errors in data model
+ignore = E501, W503, E701
+exclude = nautobot_circuit_maintenance/__init__.py
```

Comment:
That's a fair amount of code to just disable flake8 for across the board - same here, can we add targeted `# noqa` to specific lines instead of excluding the whole file?

Reply:
This list was quite larger number. Trying to get this squared away.